django_freeradius/base/models.py

```diff
@@ -399,11 +399,26 @@ class AbstractRadiusAccounting(models.Model):
     framed_ip_address = models.GenericIPAddressField(verbose_name=_('framed IP address'),
                                                      db_column='framedipaddress',
                                                      db_index=True,
-                                                     # the default MySQL freeradius schema defines
-                                                     # this as NOT NULL but defaulting to empty string
-                                                     # but that wouldn't work on PostgreSQL
                                                      null=True,
                                                      blank=True)
+    framed_ipv6_address = models.GenericIPAddressField(verbose_name=_('framed IPv6 address'),
+                                                       db_column='framedipv6address',
+                                                       db_index=True,
```

Comment:
let's remove `db_index=True` from `framed_ipv6_address` but also `framed_address` since it's not in the original schema and the index is not being used right now by any query in the application.